frontend/ui/widget/doublespinwidget.lua

```diff
@@ -244,15 +249,13 @@ function DoubleSpinWidget:update()
             widget_title,
             widget_line,
             widget_info,
-            VerticalSpan:new{ width = Size.span.vertical_large },
             CenterContainer:new{
                 dimen = Geom:new{
                     w = self.width,
-                    h = widget_group:getSize().h,
+                    h = widget_group:getSize().h + 4 * Size.padding.large,
```

Comment:
Just moving the padding inside ? Any actual visible change ?

Reply:
https://github.com/koreader/koreader/blob/c7a3c17ce06668137e6bcb231315937a401e5abc/frontend/ui/widget/spinwidget.lua#L201
The gap between the numberpicker and buttons. Now it is the same in SW and DSW.